components/Login.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+import Image from 'next/image';
+import { login } from '../app/(auth)/login/actions';
+import Logo from '@/public/images/logo-placeholder-image.png';
+import Link from 'next/link';
+import { LoginSchema } from '@/lib/schemas';
+import { useToast } from './ui/use-toast';
+
+const Login = () => {
+  const { toast } = useToast();
+  const validateAndLoginUser = (formData: FormData) => {
+    const userData = {
+      email: formData.get('email') as string,
+      password: formData.get('password') as string,
+    };
+
+    const result = LoginSchema.safeParse(userData);
+
+    if (result.success === false) {
+      let errorMessage: string = '';
+      result.error.issues.forEach((issue) => {
+        errorMessage = issue.message;
+      });
+      toast({
+        variant: 'destructive', // no fucking idea why this shit don't work... I copied it from documentation
+        title: 'Something goes wrong...',
+        description: errorMessage,
+      });
+      return;
```

Comment:
why loop through every issue if we end up with the last one?
```suggestion
      const errorMessage = result.error.issues.at(-1)?.message;
      toast({
        variant: 'destructive', // no fucking idea why this shit don't work... I copied it from documentation
        title: 'Something went wrong...',
        description: errorMessage,
      });
```
couldn't that work?

Reply:
Yup. I agree. Changed